.changeset/tidy-rivers-sin.md

```diff
@@ -0,0 +1,6 @@
+---
+"@hashicorp/design-system-components": minor
+---
+
+- `Hds::Table` - updated `@columns` object to support `isVisuallyHidden` argument
+- `Hds::Table::Th` - updated sub-component to support `isVisuallyHidden` argument
```

Comment:
This reads to me as _we've updated `Hds::Table::Th`s subcomponent to support `isVisuallyHidden` argument_, but it may be just me.